map/place_page_info.cpp

```diff
@@ -252,4 +260,35 @@ vector<ads::Banner> Info::GetBanners() const
 
   return m_adsEngine->GetBanners(m_types, m_topmostCountryIds, languages::GetCurrentNorm());
 }
+
+namespace rating
+{
+std::string const kEmptyRatingSymbol = "-";
```

Comment:
If you don't plan to expose this constant to the rest of the project, please, put it in an anonymous namespace.

Reply:
yes, I got you